x-pack/plugins/observability/public/pages/slo_edit/components/slo_edit_form_footer.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useCallback, useMemo } from 'react';
+import { useFormContext } from 'react-hook-form';
+import { InPortal } from 'react-reverse-portal';
+import { sloEditFormFooterPortal } from '../shared_flyout/slo_add_form_flyout';
+import { paths } from '../../../../common/locators/paths';
+import { useCreateSlo } from '../../../hooks/slo/use_create_slo';
+import { useUpdateSlo } from '../../../hooks/slo/use_update_slo';
+import { useCreateRule } from '../../../hooks/use_create_rule';
+import { BurnRateRuleParams } from '../../../typings';
+import { useKibana } from '../../../utils/kibana_react';
+import { createBurnRateRuleRequestBody } from '../helpers/create_burn_rate_rule_request_body';
+import {
+  transformCreateSLOFormToCreateSLOInput,
+  transformValuesToUpdateSLOInput,
+} from '../helpers/process_slo_form_values';
+import { CreateSLOForm } from '../types';
+import { EquivalentApiRequest } from './common/equivalent_api_request';
+import { SLOInspectWrapper } from './common/slo_inspect';
+
+export interface Props {
+  slo?: GetSLOResponse;
+  onSave?: () => void;
+}
+
+export const maxWidth = 775;
+
+export function SloEditFormFooter({ slo, onSave }: Props) {
+  const {
+    application: { navigateToUrl },
+    http: { basePath },
+  } = useKibana().services;
+  const isEditMode = slo !== undefined;
+
+  const { getValues, trigger } = useFormContext<CreateSLOForm>();
+
+  const { mutateAsync: createSlo, isLoading: isCreateSloLoading } = useCreateSlo();
+  const { mutateAsync: updateSlo, isLoading: isUpdateSloLoading } = useUpdateSlo();
+  const { mutateAsync: createBurnRateRule, isLoading: isCreateBurnRateRuleLoading } =
+    useCreateRule<BurnRateRuleParams>();
+
+  const navigate = useCallback(
+    (url: string) => setTimeout(() => navigateToUrl(url)),
+    [navigateToUrl]
+  );
+
+  const isFlyout = Boolean(onSave);
+
+  const handleSubmit = useCallback(async () => {
+    const isValid = await trigger();
+    if (!isValid) {
+      return;
+    }
+
+    const values = getValues();
+
+    if (isEditMode) {
+      const processedValues = transformValuesToUpdateSLOInput(values);
+      await updateSlo({ sloId: slo.id, slo: processedValues });
+      navigate(basePath.prepend(paths.observability.slos));
+    } else {
+      const processedValues = transformCreateSLOFormToCreateSLOInput(values);
+      const resp = await createSlo({ slo: processedValues });
+      await createBurnRateRule({
+        rule: createBurnRateRuleRequestBody({ ...processedValues, id: resp.id }),
+      });
+      if (onSave) {
+        onSave();
+      } else {
+        navigate(basePath.prepend(paths.observability.slos));
+      }
+    }
+  }, [
+    basePath,
+    createBurnRateRule,
+    createSlo,
+    getValues,
+    isEditMode,
+    navigate,
+    onSave,
+    slo?.id,
+    trigger,
+    updateSlo,
+  ]);
+
+  const content = useMemo(
+    () => (
+      <EuiFlexGroup justifyContent="spaceBetween" gutterSize="s">
```

Comment:
I guess that `justifyContent="spaceBetween"` causes this weird spacing. Can you remove this? 
<img width="1502" alt="Screenshot 2024-02-22 at 11 24 34" src="https://github.com/elastic/kibana/assets/2852703/08343e4f-3d5b-4d38-aa53-f0fc130b3dd2">

Reply:
Actually `justifyContent="flexStart"` fixed that for me. Removing `justifyContent="spaceBetween"` didn't do anything